### DISCUSSION TITLE
Mono theme + Feather icons in menu

### DISCUSSION THREAD

**Discussion by gpg342** [GetPublii/Publii#1375] [General] [answered]
When I add [fi=bell] to a menu name an additional     ">   prepends the menu name and icon.  This only appears if I use the [fi=bell] shortcut.

eg.  "> Policies +icon

If I change the theme it displays just the icon as anticipated.  I have not customised / edited the Mono theme.

Any ideas / help greatly recieved.

Glen

**Comment by bobmitro** [comment]
There is only one suggestion: read the documentation.

**Reply by gpg342** [reply]
Appreciate that - i have ;-)

Literally adding Policies [fi=bell]   as the menu name -- displays correctly in other themes.  In Mono it renders with an additional "<  shown on screen before the name and icon. With no other changes than swapping to a different theme it renders correctly.

;-)

G

**Reply by bobmitro** [reply, marked as answer]
Yeap, this theme has a characteristic menu structure with a tooltip, so the icon must be added in a different way.  what was described in detail in the documentation.  so why waste time when you can read the documentation.

**Reply by gpg342** [reply]
Apologies - didn't mean to antagonize you.  Will double check the documentation as I clearly missed it.

Cheers
G

**Reply by gpg342** [reply]
Clearly I didn't read the whole document!!

Lesson learned.
G

**Reply by bobmitro** [reply]
No harm done; it's important to focus our efforts on moving forward.
